src/excmds.ts

```diff
@@ -2396,11 +2396,18 @@ export function version() {
     fillcmdline_notrail(TRI_VERSION)
 }
 
-/** Example:
-        - `mode ignore` to ignore all keys.
-
-    Note that when in ignore mode, Tridactyl will not switch to insert mode when focusing text areas/inputs. This is by design.
-*/
+/**
+ *  Switch mode.
+ *
+ *  For now you probably shouldn't manually switch to other modes than `normal` and `ignore`.
```

Comment:
Probably a bad idea to suggest that people try `mode ignore` as it's actually quite hard to get out of - should at least mention how to get out of it by default.